stock_move_change_source_location/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_move_change_source_location_wizard,access_stock_move_change_source_location_wizard,model_stock_move_change_source_location_wizard,base.group_user,1,0,0,0
```

Comment:
Could you change this to stock.group_stock_user,1,1,1,0

Reply:
Done, thanks!